src/current/v25.1/time.md

```diff
@@ -191,23 +191,6 @@ ALTER TABLE
 (2 rows)
 ~~~
 
-{{site.data.alerts.callout_info}}
-If a non-default precision level has already been specified, you cannot change the precision to a lower level.
```

Comment:
@spilchen this section about downgrading precision does not appear to be true anymore based on manual testing in v24.1+ 

i did some searching but could not find a Github or Jira issue that mentions this limitation being removed, do you know offhand what the code change / GH issue is?  if not i can dig some more

Reply:
I think it's this one: https://github.com/cockroachdb/cockroach/issues/127014. That added the ability to rewrite of a type change in the DSC, and the time was included with that I believe.

Reply:
thank you!